tox.ini

```diff
@@ -12,7 +12,7 @@ deps =
 [testenv:coverage]
 usedevelop = true
 basepython =
-    python2.7
+    python3.7
```

Comment:
Often the choice of Python 2 vs Python 3 in the coverage environment is to get 100% coverage and depends on where `# pragma: no cover` directives are found in the code. Sometimes changing from 2 to 3 requires moving such directives (hopefully not adding them). For example, this code would not be covered under Python 3 anymore:
```python
try:
    unicode
except NameError: # pragma: no cover
    # py 3
    text_type = str
else:
   # This code is not covered now
   text_type = unicode
```

But I'm assuming you know this and ran `tox -e coverage` to verify everything was still at 100% before changing 😄

Reply:
Huh, coveralls claims we still have 100% coverage.  How come?

Reply:
Ah, coveralls merges Python 2 and Python 3 coverage numbers together.

I seem to recall a blog post mentioning how on a recent-enough version of coverage.py you can specify different exclude pragmas for different Python versions.  Unfortunately I cannot find it.

Reply:
Coveralls merges the results of all the environments implicitly, so even if one version of python doesn't have complete coverage put together they all might. 

On the one hand that's nice, but on the other hand it means that running a single environment locally doesn't necessarily test everything you think it tests if you didn't know that's the way it works.

Reply:
I would say this is deferrable to a follow-up PR.

Reply:
> I seem to recall a blog post mentioning how on a recent-enough version of coverage.py you can specify different exclude pragmas for different Python versions.

You may be looking for https://nedbatchelder.com/blog/201802/coverage_45.html

To quote:

> There’s one new feature: configurator plug-ins, that let you run Python code at startup to set the configuration for coverage. This side-steps a requested feature to have different exclusion pragmas for different versions of Python.

> People wanted to be able to say, this line of code is excluded from coverage when run under Python 3, but not under Python 2. That sounds simple enough, but then some wanted to be able to exclude for Python 3.5, but not 3.6. Or, excluded when running under PyPy, but not under CPython.

> I could see this turning into a never-ending road of finer and finer differentiation. Next would be operating systems, or versions of Django, or, etc, etc, etc.

> Rather than me building all that into coverage itself, now you can write your own plug-in that makes all those determinations, and sets the exclude pragmas as you like.